plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/wrapper/xen610/XenServer610MigrateWithStorageReceiveCommandWrapper.java

```diff
@@ -56,7 +55,7 @@
     public Answer execute(final MigrateWithStorageReceiveCommand command, final XenServer610Resource xenServer610Resource) {
         final Connection connection = xenServer610Resource.getConnection();
         final VirtualMachineTO vmSpec = command.getVirtualMachine();
-        final List<Pair<VolumeTO, StorageFilerTO>> volumeToFiler = command.getVolumeToFiler();
+        final List<Pair<VolumeTO, String>> volumeToStorageUuid = command.getVolumeToStorageUuid();
```

Comment:
I checked where this was being accessed from throughout the codebase and all hypervisors should be OK with this change.

They all seemed to only care about a string UUID in the StorageFilerTO object.

This PR is only for XenMotion and vMotion. This PR does not enabled cross-cluster storage migration (for managed storage) on any other hypervisor(s) in CloudStack.